apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -0,0 +1,193 @@
+import { html, HtmlSafeString } from '@prairielearn/html';
+import { EncodedData } from '@prairielearn/browser-utils';
+import { CourseInstance } from '../lib/db-types';
+import { QuestionsPageDataAnsified } from '../models/questions';
+import { nodeModulesAssetPath, compiledScriptTag, compiledStylesheetTag } from '../lib/assets';
+
+function importBootstrapTable() {
+  // Importing javascript using <script> tags as below is *not* the preferred method, it is better to directly use 'import'
+  // from a javascript file. However, bootstrap-table is doing some hacky stuff that prevents us from importing it that way
+  return html`
+    <script src="${nodeModulesAssetPath('bootstrap-table/dist/bootstrap-table.min.js')}"></script>
+    <script src="${nodeModulesAssetPath(
+        'bootstrap-table/dist/extensions/sticky-header/bootstrap-table-sticky-header.min.js',
+      )}"></script>
+    <script src="${nodeModulesAssetPath(
+        'bootstrap-table/dist/extensions/filter-control/bootstrap-table-filter-control.min.js',
+      )}"></script>
+  `;
+}
+
+export function QuestionsTableHead() {
+  return html`
+    ${compiledScriptTag('instructorQuestionsClient.ts')}
+    ${compiledStylesheetTag('questionsTable.css')} ${importBootstrapTable()}
```

Comment:
There's no reason for `importBootstrapTable()` to be separate, as it's never used independently of `QuestionsTableHead`.

Reply:
I thought about that but I ended up leaving it since its nice to isolate the code that we don't like so that it is clear what we don't like. But I don't really care much, I can combine them if you want.

Reply:
I just went ahead and changed it.